src/types.ts

```diff
@@ -1 +1,14681 @@
-export * from "./core/input_file.ts";
+// === HELPER TYPES ===
+/** Object with no keys */
+export type Empty = Record<never, never>;
```

Comment:
Should this type be changed to
```suggestion
/** Object with no keys */
export type Empty = Record<string, never>;
```
Not that it changes much, but technically anything can be assigned to a `Record<never, never>` even numbers or strings, while only `{}` can be assigned to a `Record<string, never>`.